src/styles/product-card.scss

```diff
@@ -22,6 +22,13 @@
   &__image {
     width: 160px;
     height: 134px;
+    border: 3px solid transparent;
+    transition: all 1s, border 0s;
```

Comment:
don't use `all`, define property name to animate
what does 0s means? without transition?

Reply:
In that case 0s meant that border: 3px solid #f3f3f3 appeared immediately, but after altering 'transition: all' to definite property according to your advice, there had been no need to stick to that solution.